DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj

```diff
@@ -1127,16 +1127,13 @@
     <Content Include="App_Data\Sitefinity\Export\DYSAC\DYSAC.nuspec" />
     <Content Include="App_Data\Sitefinity\Export\DYSAC\install.ps1" />
     <Content Include="App_Data\Sitefinity\Export\DYSAC\uninstall.ps1" />
-    <Content Include="App_Data\Sitefinity\AddOns\DYSAC\DYSAC.nuspec">
-      <SubType>Designer</SubType>
-    </Content>
-    <Content Include="App_Data\Sitefinity\Export\DYSAC.1.0.0.12.nupkg" />
-    <Content Include="App_Data\Sitefinity\Export\DYSAC.1.0.0.13.nupkg" />
     <Content Include="App_Data\Sitefinity\AddOns\DYSAC\Dynamic modules\Discover Your Skills And Careers\Structure\widgetTemplates.sf" />
     <Content Include="App_Data\Sitefinity\AddOns\DYSAC\Dynamic modules\Discover Your Skills And Careers\Structure\version.sf" />
     <Content Include="App_Data\Sitefinity\AddOns\DYSAC\Dynamic modules\Discover Your Skills And Careers\Structure\Discover Your Skills And Careers.sf" />
     <Content Include="App_Data\Sitefinity\AddOns\DYSAC\Dynamic modules\Discover Your Skills And Careers\Structure\configs.sf" />
     <Content Include="App_Data\Sitefinity\AddOns\DYSAC\addons-registry.psm1" />
+    <Content Include="App_Data\Sitefinity\Export\DYSAC.1.0.0.16.nupkg" />
+    <Content Include="App_Data\Sitefinity\AddOns\DYSAC\DYSAC.nuspec" />
```

Comment:
References to the old dysac nuget packages need to be kept in project file as the rest of the Dev nuget packages

Reply:
Done, The `16` package is kept in the repository.